Ginger/Ginger/UserControlsLib/TextEditor/Gherkin/GherkinPage.xaml.cs

```diff
@@ -120,7 +120,7 @@ private void Tags_CollectionChanged(object sender, System.Collections.Specialize
                 if (guid != Guid.Empty)
                     mSolTags.Add(guid);
             }
-            ARP.xActivitiesRepositoryGrid.Tags = mSolTags;
+            ARP.xActivitiesRepositoryListView.Tags = mSolTags;
```

Comment:
GherkinPage changes? need explanation

Reply:
The unGrid control (xActivitiesRepositoryGrid) has been replaced by ucListView control (xActivitiesRepositoryListView)